sandhisplitter/tests/test_model.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from sandhisplitter.model import Model
+from testtools import TestCase
+from sandhisplitter.util import extract
+
+
+class TestModel(TestCase):
+    def setUp(self):
+        super(TestModel, self).setUp()
+        self.testModel = Model(depth=3, skip=1)
+
+    def test_load(self):
+        entries = [
+                u'വിഡ്‌ഢിയുണ്ടെങ്കില്‍=വിഡ്‌ഢി+ഉണ്ട്+എങ്കില്‍|7,11',
+                u'അകല്‍ച്ചയുടെസ്ഥാനത്തു്=അകല്‍ച്ചയുടെ+സ്ഥാനത്തു്|10',
```

Comment:
Yeah, I think we should. These were select samples to get maximum coverage.
Now that I see`model.py` and `postprocessor.py` using similar testcases, its better to have them in one place. 

Will add this in the next PR.